src/components/RightPane/SectionTable/SectionTableButtons.tsx

```diff
@@ -91,6 +91,30 @@ export const ScheduleAddCell = withStyles(styles)((props: ScheduleAddCellProps)
         }
     };
 
+    const addCourseHandler = () => {
+        closeAndAddCourse(scheduleNames.length, true);
+    };
+
+    const closeCopyAndAlert = () => {
+        const url = new URL(window.location.href);
+        const urlParam = new URLSearchParams(url.search);
+        // Reset these params in url becasue when copy a specific class's link, it only copy its course code
+        // if there is random value let in the url, it will mess up the url copied.
+        const fieldsToReset = ['courseCode', 'courseNumber', 'deptLabel', 'deptValue', 'GE', 'term'];
```

Comment:
You can move this to the top level of the file so it doesn't have to be re-calculated per handler call.